### ISSUE TITLE
Can't log into Humble Bundle account.

### ISSUE THREAD

**Issue by WellBeSerious** [lutris/lutris#5612]
### Bug description

* Sorry if this has been posted before. If so, please append this to the proper post/thread, as this may help. Thank you.

Can't log into Humble Bundle account - directly or via Lutris workaround (opening Firefox, installing Export Cookies extension, exporting humblebundle.com cookies, pasting output into Lutris). Yes I enabled showing Humble Bundle in preferences.

### How to Reproduce

1. Open Lutris.
2. (Left) click Humble Bundle.
3. (Right of it) click user picture/login.
4. Text appears showing: Humble Bundle is resticting API calls from software like Lutris and GameHub. Authentication to the service will likely fail. There is a workaround involving copying cookies from Firefox, do you want to do this instead?
5. Click Yes.
6. Window appears showing: Humble Bundle Authentication via cookie import. (I'm not writing the rest of that, you know what it says).
7. Click URL: https://addons.mozilla.org/en-US/firefox/addon/export-cookies-txt/
8. (Firefox) click Add to Firefox, Add, Okay.
9. (Top) click +/new tab, go to: https://www.humblebundle.com/
10. Sign into account.
11. Click Export Cookies extension.
12. Click "Prefix HttpOnly cookies", "humblebundle.com", saved to Desktop.
13. Opened "cookies-www-humblebundle-com.txt" from Desktop.
14. Copied all and pasted into Lutris box "Humble Bundle Authentication via cookie import": # Netscape HTTP Cookie File

www.humblebundle.com	FALSE	/	FALSE	0	usprivacy	1YYY
www.humblebundle.com	FALSE	/	FALSE	0	hbflash	None
15. Click OK.
16. Text appears showing: "Lutris has encountered an error" / "Access to https://www.humblebundle.com/api/v1/user/order denied" / Error details: (below).

### Expected behavior

Can't log into Humble Bundle account. Not directly or via Lutris workaround.

### Log output

```shell
( "Lutris has encountered an error" / "Access to https://www.humblebundle.com/api/v1/user/order denied" )

Access to https://www.humblebundle.com/api/v1/user/order denied

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/lutris/util/http.py", line 97, in _request
    request = urllib.request.urlopen(req, timeout=self.timeout)  # pylint: disable=consider-using-with

  File "/usr/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)

  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)

  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(

  File "/usr/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)

  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)

  File "/usr/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)

urllib.error.HTTPError: HTTP Error 401: Unauthorized


The above exception was the direct cause of the following exception:


Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/lutris/util/jobs.py", line 30, in target
    result = self.function(*args, **kwargs)

  File "/usr/lib/python3/dist-packages/lutris/services/base.py", line 142, in do_reload
    self.load()

  File "/usr/lib/python3/dist-packages/lutris/services/humblebundle.py", line 116, in load
    library = self.get_library()

  File "/usr/lib/python3/dist-packages/lutris/services/humblebundle.py", line 160, in get_library
    for order in self.get_orders():

  File "/usr/lib/python3/dist-packages/lutris/services/humblebundle.py", line 184, in get_orders
    gamekeys = self.make_api_request(self.api_url + "api/v1/user/order")

  File "/usr/lib/python3/dist-packages/lutris/services/humblebundle.py", line 134, in make_api_request
    request.get()

  File "/usr/lib/python3/dist-packages/lutris/util/http.py", line 137, in get
    return self._request("GET", data)

  File "/usr/lib/python3/dist-packages/lutris/util/http.py", line 100, in _request
    raise UnauthorizedAccessError("Access to %s denied" % self.url) from error

lutris.util.http.UnauthorizedAccessError: Access to https://www.humblebundle.com/api/v1/user/order denied
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: NO
Mangohud: NO
Gamemode: YES
Steam: YES
In Flatpak: NO
[System]
OS: Linux Mint 21.3 Virginia
Arch: x86_64
Kernel: 5.19.0-50-generic
Desktop: MATE
Display Server: x11
[CPU]
Vendor: AuthenticAMD
Model: AMD Ryzen 5 5600X 6-Core Processor
Physical cores: 6
Logical cores: 12
[Memory]
RAM: 62.7 GB
Swap: 7.6 GB
[Graphics]
Vendor: NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce RTX 3050/PCIe/SSE2
OpenGL Version: 4.6.0 NVIDIA 550.90.07
OpenGL Core: 4.6.0 NVIDIA 550.90.07
OpenGL ES: OpenGL ES 3.2 NVIDIA 550.90.07
Vulkan Version: 1.3.204
Vulkan Drivers: NVIDIA GeForce RTX 3050 (1.3.277)
```

**Comment by WellBeSerious** [comment]
Update: I also sent Humble Bundle a request to fix this bug/authentication issue.

**Comment by WellBeSerious** [comment]
Update 3: Turns out that extension has these entries (only appearing when on that site): "www.humblebundle.com" and "humblebundle.com". I opened both. "humblebundle.com" one is much longer, and has code which may include my login info, so I won't post here. I pasted that one into Lutris' box, AND IT LOGGED IN! Lustis/Humble shows 21 entries (games and software, which is fine considering Lutris doesn't know the difference). I have many entries on my Humble download page, but only ones with download appear in Lutris/Humble, which makes sense since keys expire. It works, it's solved.
